src/sentry/digests/notifications.py

```diff
@@ -42,7 +35,7 @@ def split_key(key: str) -> Tuple["Project", "ActionTargetType", Optional[str]]:
 
 
 def unsplit_key(
-    project: "Project", target_type: ActionTargetType, target_identifier: Optional[str]
```

Comment:
Is there a functional difference between `Optional[str]` and `str | None`?

Reply:
`Optional[str]` is equivalent to `Union[str, None]` which is equivalent to `str | None`.